src/Postgres/LoadTypes.idr

```diff
@@ -68,7 +67,7 @@ oidTypeStrings : List String
 oidTypeStrings = ["oid"]
 
 quote : String -> String
-quote str = '\'' <+ str +> '\''
+quote str = (strCons '\'' str) ++ "\\"
```

Comment:
This used to put single quotes around the string but you've changed the closing quote to a backslash.